CHANGES_NEXT_RELEASE

```diff
@@ -25,6 +25,7 @@ Add:  URI parameter 'q' for GET /v2/entities (Issue #967)
 Add:  PUT /v2/entities/<entityId>/attrs/<attrName>/value (Issue #999)
 Add:  DELETE /v2/entities/<entityId>/attrs/<attrName> (Issue #993)
 Add:  PUT /v2/entities/<entityId>/attrs/<attrName> (Issue #991)
-Add:  GET /v2/types/<type> (Issue #1028)
+Add:  GET /v2/type/<type> (Issue #1028)
```

Comment:
In https://docs.google.com/spreadsheets/d/1f4m624nmO3jRjNalGE11lFLQixnfCENMV6dc54wUCCg/edit#gid=50130961 the URl are rigth.